kedro-datasets/setup.py

```diff
@@ -216,14 +215,12 @@ def _collect_requirements(requires):
     "requests~=2.20",
     "ruff~=0.0.290",
     "s3fs>=0.3.0, <0.5",  # Needs to be at least 0.3.0 to make use of `cachable` attribute on S3FileSystem.
-    "snowflake-snowpark-python~=1.0.0; python_version == '3.8'",
```

Comment:
This will need to be brought back for the docs to build, for example

```
"snowflake-snowpark-python~=1.0"
```

Reply:
yes, thanks for help, @astrojuanlu !